WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Product Details/AggregatedProductListViewController.swift

```diff
@@ -0,0 +1,155 @@
+import UIKit
+import Yosemite
+
+
+/// A simple Product List presented from the Order Details screen -> Shipping Labels Packages.
+///
+/// The list shows the product name and the quantity.
+///
+final class AggregatedProductListViewController: UIViewController {
+    @IBOutlet private var tableView: UITableView!
+
+    private var viewModel: OrderDetailsViewModel
+    private var products: [Product]? = []
+    private var items: [AggregateOrderItem]
+
+    private let imageService: ImageService = ServiceLocator.imageService
```

Comment:
nit: A small improvement would be to inject this with a default parameter, so we make dependencies clear.

Reply:
Done ✅